devfiles/che-theia-generator-cli.devfile.yaml

```diff
@@ -0,0 +1,98 @@
+# A devfile to work on the che-theia generator cli: che:theia init, che:theia production, etc ...
+
+specVersion: 0.0.1
+name: cheTheiaGenerator-in-che
+projects:
+  - name: che-theia
+    source:
+      type: git
+      location: 'https://github.com/eclipse/che-theia.git'
+  - name: theia
+    source:
+      type: git
+      location: 'https://github.com/theia-ide/theia.git'
+  - name: che-theia-generator
+    source:
+      type: git
+      location: 'https://github.com/ws-skeleton/che-theia-generator.git'
+components:
+  - 
+    alias: git-devtools
+    type: dockerimage
+    image: sunix/git-devtools
```

Comment:
We mustn't use personal images in Eclipse projects. Please move the image to Eclipse organization.

Reply:
Why not ? These are not binaries, what about source code like spring petclinic you clone, what about all the components that are vscode extension and not from Eclipse ?

Reply:
I am not sure that everyone need to use this container. Maybe don't include this container, and it will save a bit ram.

Reply:
@sunix it is not appropriate for organization like Eclipse.

> These are not binaries

No, with docker image you bring binaries, which is build and stored in your personal account.

Reply:
1. This is not bringing third party dependencies to the the project.
2. It means that you can not use yarn or npm or git or maven or docker because these are not Eclipse. What if you have a script in che-theia repo that invoking docker build. Docker is not a Eclipse project so it can't be in a script in che-theia repo ?
3. You don't use git from the terminal ?

but I am ok to move it. Just your reasons don't make sense.

Reply:
1. How is this related to binaries inside personal docker image?
2. You didn't understand me. I am not talking about using tools, my concern is about image origin.
3. I use git from terminal, but so far haven't use `sunix/git-devtools` because git included in other containers.

About reasons. Think about it from another side: why do we have all images in Eclipse organization and not in personal accounts? Why do we need Eclipse namespace on dockerhub? I hope you'll understand the problem, so your arguments doesn't make sense.

Reply:
For me, we should not depend on personal github repositories in our public project.
And if we put some devfile which depends on your docker image we should put dockerfile for that image in this repository. It's a matter of maintainability of that devfile and docker image.

Reply:
so in our devfiles we should not use any extensions from vscode marketplace because of maintainability, etc ... let's close this issue for the time being because we have plenty of other things to deal with then debating ...